crates/string_wizard/src/magic_string/mod.rs

```diff
@@ -40,6 +40,24 @@ pub struct MagicString<'s> {
   last_searched_chunk_idx: ChunkIdx,
 }
 
+impl Default for MagicString<'_> {
+  fn default() -> Self {
+    MagicString {
```

Comment:
If you really need a default impl, I will suggest constructing the instance using `MagicString::new("")` instead of filling up these fields manually to prevent potential invalid states.